js/templates/modals/editListing/variant.html

```diff
@@ -1,20 +1,22 @@
-<div class="col6 simpleFlexCol">
```

Comment:
Can we please keep the simpleFlexCols? They are need so that fields remain bottom aligned when only one has an error:

![image](https://cloud.githubusercontent.com/assets/794517/24157214/023ddefc-0e16-11e7-8404-e293284b6bc1.png)